### PR TITLE
Fix marquee label not working with layout constraints

### PR DESCRIPTION
**Pull request by flambert**

### CODE REVIEW

File: Pod/Classes/MRGMarqueeLabel.m [hunk 1]
```diff
@@ -95,43 +85,66 @@ - (CGSize)intrinsicContentSize {
 
 - (void)layoutSubviews {
     [super layoutSubviews];
-    [self layoutLabels];
+    
+    if (self.needsAnimationReset || !CGRectEqualToRect(self.previousRect, self.bounds)) {
```
Comment (simondec) [inline Pod/Classes/MRGMarqueeLabel.m]: You could have compared `self.bounds` to `self.contentView.frame` instead of adding a private `CGRect` variable?

Comment (flambert) [reply Pod/Classes/MRGMarqueeLabel.m]: Good idea

**Comment by simondec** [comment]
There could have been a slight optimization to prevent frames resetting when the only changes are animation parameters (could have just resetted the core animation).  But its all good 👍

**Comment by flambert** [comment]
Opened https://github.com/mirego/MRGMarqueeLabel/issues/3 enhancement issue to handle that later.

**Comment by simondec** [comment]
👍

**Pull request merged**